spring-vault-core/src/main/java/org/springframework/vault/core/VaultTransitOperations.java

```diff
@@ -123,6 +124,16 @@
 	String encrypt(String keyName, byte[] plaintext, VaultTransitContext transitRequest);
 
 	/**
```

Comment:
these are the first version of encrypt/decrypt APIs which accept the List of String. However we can add List/Array of byte[] version of the APIs and also the one which takes VaultTransitContext.
Once this PR is accepted, I can create ticket for these flavors of the APIs and work on for the next snapshot version.

let me know your thought.
thanks.